contracts/wallet/Proxy.sol

```diff
@@ -24,7 +24,7 @@ pragma solidity ^0.6.12;
  */
 contract Proxy {
 
-    address implementation;
+    address immutable public implementation;
```

Comment:
Just a reminder to remove that ignore from the `slither` db as it's been asking us to do this.